Pure_Parallels/update_parallel_wcs.py

```diff
@@ -0,0 +1,155 @@
+import os, sys
```

Comment:
imports should be on separate lines specified by PEP8 standards. 

import os
import sys

Reply:
done